leanplum_data_export/schemas/states.schema.json

```diff
@@ -0,0 +1,30 @@
+[
+  {
```

Comment:
There aren't any actual state values in leanplum yet so some of these types are my best guess